pkg-manager/resolve-dependencies/src/nodeIdUtils.ts

```diff
@@ -18,5 +18,5 @@ export function createNodeId (parentNodeId: string, pkgId: string) {
 }
 
 export function splitNodeId (nodeId: string) {
-  return nodeId.slice(1, -1).split('>')
+  return nodeId.split('>').slice(1, -1)
```

Comment:
having a slice after splitting the string, allows copying less data, making the dependency resolution a few seconds faster on a large monorepo